packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -522,9 +522,15 @@ function getRuntime(process: NodeJS.Process, logger: AstroIntegrationLogger): Ru
 		);
 		return 'nodejs18.x';
 	}
-	if (support.status === 'current') {
+	if (support.status === 'default') {
 		return `nodejs${major}.x`;
 	}
+	if (support.status === 'retiring') {
+	  logger.info(
+      `Your project is being built for Node.js ${major} as the runtime, which is retiring by ${support.removal}.`
+    );
+    return `nodejs${major}.x`;
+	}
```

Comment:
I don't think we need to log this because there is no user action to be taken. 18 has a long road of support ahead of it.

Reply:
Well for security support it does, which ends in a year (https://endoflife.date/nodejs)

Reply:
I agree with Arsh, 18 is still under maintenance LTS so it should be fine to keep using it. I think it's helpful to have a retiring status, but maybe only warn if it's around 3 months away?

Reply:
Like this? https://github.com/withastro/astro/pull/10611/commits/5131f126d3a9cf23a3f573f56e7bd7e3155b501e